cluster/examples/kubernetes/minio/operator.yaml

```diff
@@ -73,12 +73,17 @@ subjects:
   name: rook-minio-operator
   namespace: rook-minio-system
 ---
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: rook-minio-operator
   namespace: rook-minio-system
+  labels:
+    app: rook-minio-operator
 spec:
+  selector:
+    matchLabels:
+      app: rook-minio-operator
```

Comment:
What is the purpose of the selectors? To me it looks like duplication.

Reply:
Kubernetes in apps/v1 requires Deployments, StatefulSets and so on to have this `selector`.
There isn't too much I found while searching a "because of this", at least the API reference says for Deployments:
> Label selector for pods. Existing ReplicaSets whose pods are selected by this will be the ones affected by this deployment. It must match the pod template's labels.
> - https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.13/#deploymentspec-v1-apps

Reply:
+1 I believe the rook-ceph-operator should have it, at least it seems to be expected (OLM does for instance).

Reply:
Oops you did it already!